lib/android-helpers.js

```diff
@@ -472,6 +472,15 @@ helpers.pushSettingsApp = async function (adb, throwError = false) {
     return;
   }
 
+  if (await adb.getApiLevel() < 23) { // Android 6- devices should granted permissions
+    // Making sure granting android.permission.SET_ANIMATION_SCALE
+    // Read Read https://github.com/appium/appium/pull/11640#issuecomment-438260477
+    logger.info('Granting android.permission.SET_ANIMATION_SCALE, CHANGE_CONFIGURATION, ACCESS_FINE_LOCATION by pm grant');
+    await adb.grantPermission(SETTINGS_HELPER_PKG_ID, 'android.permission.SET_ANIMATION_SCALE');
```

Comment:
It is expensive to call adb shell for each permission. I'll add a shortcut to appium-adb to be able to grant multiple permissions within single shell call

Reply:
Is https://github.com/appium/appium-adb/pull/381 expected way? (I've just created)